content/en/apps/tutorials/multi-facility-users.md

```diff
@@ -0,0 +1,201 @@
+---
+title: "Setting up Multi-facility users"
+linkTitle: Multi Facility Users
+weight: 6
+description: >
+  Creating and assigning users to multiple places in the CHT UI
+relatedContent: >
+  apps/concepts/users
+  apps/reference/app-settings
+  apps/tutorials/contact-and-users-1
+---
+
+{{% pageinfo %}}
+This tutorial will take you through how to create and assign users to multiple places in the CHT UI, including;
+
+- Create contacts and their associated users
+- Create places and assign contacts to those places
+- Assigning users to multiple places
+
+App settings allow you to both persist information that is critical to the application outside the code, and to create profiles that store the preferences for project deployments.
```

Comment:
Do you mean the admin app? 

```suggestion
The [App Management]({{< relref "apps/features/admin" >}}) allows you to both persist information that is critical to the application outside the code, and to create profiles that store the preferences for project deployments.
```

Or the actual app-settings file? 

```suggestion
The [CHT application settings]({{< relref "apps/tutorials/application-settings" >}}) allows you to both persist information that is critical to the application outside the code, and to create profiles that store the preferences for project deployments.
```

Reply:
I meant the app-settings file.